.github/actions/before-install/action.yml

```diff
@@ -36,13 +36,13 @@ runs:
         echo "Setting up CI to run with commit flag [affected:*] flag."
         echo "BREAK_ACTION=true" >> $GITHUB_ENV
 
-    - name: ci:force flag parser
-      shell: bash
-      if: ${{ contains(github.event.head_commit.message , '[ci:force]') }}
-      run: |
-        echo "Forcing CI to run with commit flag [ci:force]."
-        echo CI_FORCE_RUN=true >> $GITHUB_ENV
-        echo "BREAK_ACTION=true" >> $GITHUB_ENV
+    # - name: ci:force flag parser
```

Comment:
I am not sure is the final status od the PR but in the case better to not leave commented code